frame/pablo/rpc/src/lib.rs

```diff
@@ -118,4 +126,25 @@ where
 			)))
 		})
 	}
+
+	fn redeemable_assets_for_given_lp_tokens(
+		&self,
+		pool_id: SafeRpcWrapper<PoolId>,
+		lp_amount: SafeRpcWrapper<Balance>,
+		at: Option<<Block as BlockT>::Hash>,
+	) -> RpcResult<(SafeRpcWrapper<Balance>, SafeRpcWrapper<Balance>)> {
```

Comment:
I think we should avoid creating interfaces that assume fixed number of assets in a pool while also ignoring assetIds, like 2 in this case. We can create a data structure to return here that has the assetId+balance list.